src/components/Token.vue

```diff
@@ -1,18 +1,36 @@
 <template>
-  <img
-    :src="
-      `https://raw.githubusercontent.com/balancer-labs/snapshot/develop/spaces/${space}/logo.png`
-    "
-    class="d-inline-block bg-gray-9 v-align-middle line-height-0 circle border"
-    :style="{
-      width: `${size || 22}px`,
-      height: `${size || 22}px`
-    }"
-  />
+  <span>
+    <img
+      v-if="!loadError"
+      :src="symbolUrl"
+      class="d-inline-block bg-gray-9 v-align-middle line-height-0 circle border"
+      :style="{
+        width: `${size || 22}px`,
+        height: `${size || 22}px`
+      }"
+      :onerror="onError()"
```

Comment:
With `onerror` the browser will have to send a request to check if image is available, i dont think it's necessary. We don't need to change the logic here, we can keep how it was before and just assume the logo for each strategy will be available. You can just pass a prop "scoreId" and display the logo like this:
```js
const id = this.scoreId ? this.scoreId : '';
const url = `https://raw.githubusercontent.com/balancer-labs/snapshot/develop/spaces/${space}/logo${id}.png`;
return url;
```
Then we just need to store logo.png logo1.png logo2.png etc

Reply:
Changed to:
```
logo.png -- strategy index 0
logo1.png -- strategy index 1
logo2.png -- strategy index 2
[...]
```